cluster/images/etcd-version-monitor/etcd-version-monitor.go

```diff
@@ -215,19 +320,21 @@ func main() {
 	pflag.CommandLine.AddGoFlagSet(goflag.CommandLine)
 	pflag.Parse()
 
+	for _, wl := range strings.Split(whitelisted, ",") {
+		gatherer.exported[strings.TrimSpace(wl)] = &exportedMetric{}
+	}
+
```

Comment:
Don't we need to register all other metrics?

Reply:
We might as well now thatwe have the proxying code needed to do that. Great idea actually. Will add.